scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -756,8 +764,40 @@ get_transfer()
     fi
 }
 
+check_server_ssl_config()
+{
+    local section=$1
+    local sst_mode=$2
+    ssl_ca=$(parse_cnf $section ssl-ca "")
+    ssl_cert=$(parse_cnf $section ssl-cert "")
+    ssl_key=$(parse_cnf $section ssl-key "")
+    if [ 0 -eq $encrypt -a -n "$ssl_cert" -a -n "$ssl_key" ]
```

Comment:
this is interesting.  So if the user sets encrypt=0 (which is no encryption), then it's possible they'll get SSL encryption anyway?